### ISSUE TITLE
Needs to support command for the build step of CocosCreator (Project -> Build -> Build)

### ISSUE THREAD

**Issue by seanxlliu** [cocos2d/cocos2d-console#351]
There is no same console command for building projects like CocosCreator building step  (Project -> Build -> Build) before compiling

**Comment by hdtench** [comment]
I have the same question like you . is there any command to build CocosCreator projects  such as  "cocos run -p ios"   ???

**Comment by seanxlliu** [comment]
no, I think they don't have plan for this feature yet. we are using CocosCreator v1.1.1 now

**Comment by hdtench** [comment]
http://www.cnblogs.com/zisou/p/cocos2dx3.html   i found a way to solve the problem.  We use the same version .
